plugins/woocommerce/tests/e2e-pw/fixtures/fixtures.js

```diff
@@ -51,6 +51,88 @@ exports.test = base.test.extend( {
 		await use( wpApi );
 	},
 
+	wcbtApi: async ( { baseURL }, use ) => {
+		const wcbtApi = await base.request.newContext( {
+			baseURL,
+			extraHTTPHeaders: {
+				Authorization: `Basic ${ Buffer.from(
+					`${ admin.username }:${ admin.password }`
+				).toString( 'base64' ) }`,
+				cookie: '',
+			},
+		} );
+
+		await use( {
+			/**
+			 * Deactivate WC.
+			 */
+			deactivateWC: async () => {
+				return await wcbtApi.fetch(
+					'./wp-json/wc-admin-test-helper/live-branches/deactivate/v1',
+					{ method: 'GET' }
+				);
+			},
+
+			/**
+			 * Install the latest WC including pre-releases.
+			 */
+			installLatest: async () => {
+				const response = await wcbtApi.fetch(
+					'./wp-json/wc-admin-test-helper/live-branches/install/latest/v1',
+					{
+						method: 'POST',
+						data: { include_pre_releases: true },
+					}
+				);
+
+				if ( ! response.ok() ) {
+					throw new Error(
+						`Failed to install latest WC: ${ response.status() } ${ await response.text() }`
+					);
+				}
+
+				return await response.json();
+			},
+
+			/**
+			 * Install a specific WC version.
+			 *
+			 * @param {string} resolvedVersion - The WooCommerce version (e.g., '9.6.0-beta').
+			 */
+			installSpecificVersion: async ( resolvedVersion ) => {
+				const downloadUrl = `https://github.com/woocommerce/woocommerce/releases/download/${ resolvedVersion }/woocommerce.zip`;
+				return await wcbtApi.fetch(
+					'./wp-json/wc-admin-test-helper/live-branches/install/v1',
+					{
+						method: 'POST',
+						data: {
+							pr_name: resolvedVersion,
+							download_url: downloadUrl,
+							version: resolvedVersion,
+						},
+					}
+				);
+			},
+
+			/**
+			 * Activate a specific WC version.
+			 *
+			 * @param {string} resolvedVersion - The WooCommerce version (e.g., '9.6.0-beta').
+			 */
+			activateWC: async ( resolvedVersion ) => {
+				return await wcbtApi.fetch(
+					'./wp-json/wc-admin-test-helper/live-branches/activate/v1',
+					{
+						method: 'POST',
+						data: {
+							version: resolvedVersion,
+						},
+					}
+				);
+			},
+		} );
```

Comment:
You don't need to implement all these methods here (not even sure that works?), all you need is to use the `wcbtApi`, to expose it to wherever the fixture is used.

The code from the functions `deactivateWC`, `installLatest`, etc. should be in the `install-ws.setup.js` only. I don't think we'll need this anywhere else, so I don't think extracting the logic in public functions is required.

```suggestion
		await use( wcbtApi );
```

Reply:
Done in [de3a60a](https://github.com/woocommerce/woocommerce/pull/55063/commits/de3a60a5e850f95db30cd9b6eacb7a0648c28fea)